src/GitVersionHelper.php

```diff
@@ -39,7 +40,9 @@ public static function getVersion()
         chdir(base_path());
 
         // Get version string from git
-        $output = shell_exec('git describe --always --tags --dirty');
+        $process = new Process(['git describe', '--always', '--tags', '--dirty']);
```

Comment:
Pretty sure this'll fail saying "no such command `git describe`". I think they need to be separate parts of the array, since "describe" is the first arg to the "git" binary.

Reply:
You're right